src/HashBus.Viewer.MostMentioned/LeaderboardView.cs

```diff
@@ -89,10 +89,10 @@ static class LeaderboardView<TEntry> where TEntry : WebApi.IEntry
 
                     if (showPercentages)
                     {
-                        tokens.Add($" ({currentEntry.Count / (double)currentLeaderboard.Count:P0})".DarkGray());
+                        tokens.Add($" ({currentEntry.Count/(double) currentLeaderboard.Count:P0})".DarkGray());

```

Comment:
This (and some of the other whitespace changes) doesn't seem to be idiomatic, and also conflicts with Visual Studio's "Format Document" action.